default.nix

```
@@ -1,10 +1,14 @@
 { pkgs ? import <nixpkgs> {}
-, nodejs ? pkgs.nodejs
+, nodejs ? pkgs.nodejs-8_x
```

we should use the default version
```suggestion
, nodejs ? pkgs.nodejs
```

---

this change makes ci tests fail

```
 ✘  ~/projects/yarn2nix   pull85  export NIXPKGS_CHANNEL=nixos-18.03 && ./run-tests.sh
--- Running tests with IFD enabled
building '/nix/store/xvv3zdgw7h9q1ilikhph9wx3qhkg9kb8-fixup_yarn_lock.drv'...
installing
node -v
v6.14.4
patching script interpreter paths in /nix/store/1cga0h2vqbr7xxxvjn9ccvk4n8jzw0zp-fixup_yarn_lock
/nix/store/1cga0h2vqbr7xxxvjn9ccvk4n8jzw0zp-fixup_yarn_lock/bin/fixup_yarn_lock: interpreter directive changed from "/usr/bin/env node" to "/nix/store/85y6qr70pyv2njvg1pnsl4zcln540iyc-nodejs-6.14.4/bin/node"
building '/nix/store/x2qkrxrb97lmzzj2k840s4xnksbrsqax-yarn2nix-modules-1.0.0.drv'...
configuring
building
yarn config v1.5.1
success Set "yarn-offline-mirror" to "/nix/store/3xz5j2lp0y2cqcnhsinvhzgdcqqj5ncj-offline".
Done in 0.03s.
/nix/store/1cga0h2vqbr7xxxvjn9ccvk4n8jzw0zp-fixup_yarn_lock/bin/fixup_yarn_lock:46
        )
        ^

SyntaxError: Unexpected token )
    at createScript (vm.js:56:10)
    at Object.runInThisContext (vm.js:97:10)
    at Module._compile (module.js:549:28)
    at Object.Module._extensions..js (module.js:586:10)
    at Module.load (module.js:494:32)
    at tryModuleLoad (module.js:453:12)
    at Function.Module._load (module.js:445:3)
    at Module.runMain (module.js:611:10)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:160:9)
builder for '/nix/store/x2qkrxrb97lmzzj2k840s4xnksbrsqax-yarn2nix-modules-1.0.0.drv' failed with exit code 1
```

I propose to remove tests for NIXPKGS_CHANNEL=nixos-18.03

---

apart from that I would say the pr is ready